core/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -598,10 +590,19 @@ public Snapshot newSnapshotFromGen(long minGeneration) throws IOException {
     }
 
     public Snapshot newSnapshotFromMinSeqNo(long minSeqNo) throws IOException {
+        return newSnapshotBetweenMinAndMaxSeqNo(minSeqNo, Long.MAX_VALUE);
+    }
+
+    /**
+     * Returns a snapshot with operations having a sequence number equal or greater than <code>minSeqNo</code> and
+     * equal or lesser than <code>maxSeqNo</code>.
+     */
+    public Snapshot newSnapshotBetweenMinAndMaxSeqNo(long minSeqNo, long maxSeqNo) throws IOException {
```

Comment:
maybe name this `getSnapshotBetween(long minSeqNo, long maxSeqNo)` the min and max is implicit?